app/config/collections2.php

```diff
@@ -558,6 +558,471 @@
         ],
     ],
 
+    'projectsPlatforms' => [
```

Comment:
I think to fit with current patterns, we should omit the `projects` prefix on these tables.

Reply:
Will do. Same for filter names.